iaso/api/setup_account.py

```diff
@@ -12,6 +12,56 @@
 logger = logging.getLogger(__name__)
 
 
+DEFAULT_PERMISSIONS_FOR_NEW_SETUP_ACCOUNT_USER = [
```

Comment:
From @bmonjoie we should use constants here and i totally agree.
Also, this is not a superuser, we have to maintain this while adding new permissions.

Reply:
I reworked it so the list comes from the model file.

Reply:
@SimonDus What @beygorghor is saying is not going to be possible until my PR is merged #718